gtsam/inference/EliminateableFactorGraph.h

```diff
@@ -217,60 +215,108 @@ namespace gtsam {
 
     /** Compute the marginal of the requested variables and return the result as a Bayes net.  Uses
      *  COLAMD marginalization ordering by default
-     *  @param variables Determines the variables whose marginal to compute, if provided as an
-     *         Ordering they will be ordered in the returned BayesNet as specified, and if provided
-     *         as a KeyVector they will be ordered using constrained COLAMD.
-     *  @param function Optional dense elimination function, if not provided the default will be
-     *         used.
+     *  @param variables Determines the *ordered* variables whose marginal to compute, 
+     *         will be ordered in the returned BayesNet.
```

Comment:
I think it would be clearer as "will be ordered in the returned BayesNet _as specified_" (like in the original docstring)

Reply:
I like that, will do.